x-pack/elastic-agent/docs/inputs/input-logfile.asciidoc

```diff
@@ -0,0 +1,163 @@
+:type: logfile
+
+[id="{beatname_lc}-input-{type}"]
```

Comment:
I'd hard code the ID.

Reply:
Like  this `elastic-agent-input-logfile` ?